app/react-native/readme.md

```diff
@@ -69,7 +69,9 @@ For CRNA apps:
 For RN apps:
 
     react-native run-<platform>
-
+    
+If you are using Android and you get the following error after running the app: `'websocket: connection error', 'Failed to connect to localhost/127.0.0.1:7007'`, you have to forward the port 7007 on your device/emulator to port 7007 on your local machine with the following command:
```

Comment:
great!!! mind putting this below "once your app ..." and adding an extra line break between the paragraphs?